### PR TITLE
Ruby: Release 3.3

### PR DESCRIPTION
**Pull request by tiagogodinho**
Ref: #866

### CODE REVIEW

**Comment by tiagogodinho** [comment]
@microsoft-github-policy-service agree

> @tiagogodinho please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
>  > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

**Comment by eljog** [comment]
Thank you for your contribution. We will review it soon.
I have approved for workflow runs

**Review by eljog — COMMENTED** [review]
Changes looks good.
Added a comment on the manifest

**Review by samruddhikhandale — APPROVED** [review]
Looks good, thank you!

File: src/ruby/manifest.json [hunk 1]
```diff
@@ -1,8 +1,10 @@
 {
-	"version": "1.0.6",
+	"version": "1.0.7",
```
[the remark this answers is not in the document]
Comment (samruddhikhandale) [reply src/ruby/manifest.json:2]: Looks like merging main to this branch resolved this comment!

**Pull request merged**